### PR TITLE
Add greeting from Malena

### PR DESCRIPTION
**Pull request by mahon479**
I added Malena's greeting.

### CODE REVIEW

**Review by NicMcPhee — DISMISSED** [review]
This looks great - thanks for the help!

**Review by AndrewLam288 — DISMISSED** [review]

**Review by elyjohnson — DISMISSED** [review]

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -13,12 +13,17 @@ public String generateOutput() {
...
     builder.append(nicSaysHello());
     builder.append(patSaysHello());
 
     return builder.toString();
   }
 
+
+  private String malenaSaysHello() {
+    return "Malena says 'Hi!'\n";
+ }
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java:26]: It looks like this close brace isn't indented correctly and should be one more space to the right?

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -29,6 +34,7 @@ private String andreasSaysHello() {
   
   private String patSaysHello() {
     return "Pat says 'Hello to everyone!'\n";
+
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java:37]: I probably wouldn't have this blank line here.

File: src/main/java/hellos/Hellos.java (hunk 1 above)
Comment (NicMcPhee) [reply src/main/java/hellos/Hellos.java:26]: I would also typically put a blank line between function definitions, so add a blank line after this curly brace.

**Review by NicMcPhee — CHANGES_REQUESTED** [review]
There are a couple of minor spacing issues that you should probably fix.

Other than that it look great!

File: src/main/java/hellos/Hellos.java [hunk 3]
```diff
@@ -13,13 +13,18 @@ public String generateOutput() {
...
     builder.append(tristanSaysHello());
 
     return builder.toString();
   }
 
+
+  private String malenaSaysHello() {
+    return "Malena says 'Hi!'\n";
+ }
   private String KenSaysHello() {
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: Two minor whitespace issues:

- The close curly brace on this definition is one space too far to the left.
- You would typically put a blank line between two function definitions.

```suggestion
  private String malenaSaysHello() {
    return "Malena says 'Hi!'\n";
  }
  
  private String KenSaysHello() {
```

File: src/main/java/hellos/Hellos.java [hunk 4]
```diff
@@ -30,6 +35,7 @@ private String andreasSaysHello() {
 
   private String patSaysHello() {
     return "Pat says 'Hello to everyone!'\n";
+
   }
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: We typically wouldn't want a blank line like this at the end of a function definition.

```suggestion
  private String patSaysHello() {
    return "Pat says 'Hello to everyone!'\n";
  }
```

**Review by NicMcPhee — DISMISSED** [review]

**Review by NicMcPhee — DISMISSED** [review]

**Review by KenBroden — DISMISSED** [review]
Looks good

**Review by KenBroden — APPROVED** [review]

**Review by LinneaMay — APPROVED** [review]
looks good!

**Pull request merged**